nucypher/cli/commands/status.py

```diff
@@ -51,6 +52,22 @@
 from nucypher.config.constants import NUCYPHER_ENVVAR_PROVIDER_URI
 from nucypher.utilities.events import generate_events_csv_filepath
 
+STAKING_ESCROW = 'StakingEscrow'
+POLICY_MANAGER = 'PolicyManager'
```

Comment:
Since this is one of only two usages in the entire codebase (the other on in the status/events CLI) do you still prefer this as an imported constant?  I removed it in this PR.

Reply:
This comment, https://github.com/nucypher/nucypher/pull/2867#discussion_r812160420, was meant for here